### PR TITLE
Add the command `git: checkout current file`. #92

### LINKED ISSUE #92: Checkout current file
Hello!

The only feature I miss from SublimeGit it's the ability to checkout the current active file to HEAD without having to go to the status view.

I've implemented this in GitSavvy like this:

````python
class GsCheckoutCurrentFileCommand(WindowCommand, GitCommand):

    """
    Reset the current active file to HEAD.
    """

    def run(self):
        view_file_name = self.window.active_view().file_name()
        if view_file_name:
            self.checkout_file(view_file_name)
            sublime.status_message("Successfully checked out %s from HEAD." % view_file_name)

````

and in Default.sublime-commands:

````json
{
     "caption": "git: checkout current file",
     "command": "gs_checkout_current_file"
},
````

I didn't add `@util.actions.destructive(description="discard a file")` because it's a direct action against the file.

I don't know if this really is useful or in the roadmap, I didn't make a proper PR because I couldn't decide where to put the command and documentation (my best guess was in `checkout.py`, but I'm not sure).

Great package!

### PR DESCRIPTION
**Pull request by NicoSantangelo**
Adds a command to  checkout the current active file to HEAD without having to go to the status view (from #92 ).

For any change, let me know!

Thanks!

**Edit:** Closes #92.

### CODE REVIEW

File: core/commands/checkout.py [hunk 1]
```diff
@@ -90,3 +90,15 @@ def on_selection(self, remote_branch_index):
...
+class GsCheckoutCurrentFileCommand(WindowCommand, GitCommand):
+
+    """
+    Reset the current active file to HEAD.
+    """
+
+    def run(self):
+        if self.file_path:
+            self.checkout_file(self.file_path)
+            sublime.status_message("Successfully checked out %s from HEAD." % self.file_path)
```
Comment (divmain) [inline core/commands/checkout.py]: Nothing wrong with this - but for the sake of consistency across the project, please use `.format(self.file_path)` here.

**Comment by divmain** [comment]
Just the one comment - ping me when you made the change, and I'll merge this.  Thanks for the PR!

**Comment by NicoSantangelo** [comment]
Done! (sorry I didn't notice you where using `.format`)

**Comment by divmain** [comment]
Thanks!

**Pull request merged**